src/rust/cryptography-x509-verification/src/policy/mod.rs

```diff
@@ -329,6 +338,19 @@ impl<'a, B: CryptoOps> Policy<'a, B> {
     }
 
     fn permits_basic(&self, cert: &Certificate<'_>) -> Result<(), ValidationError> {
+        // CA/B 6.1.5: Key sizes
+        // NOTE: We don't currently enforce that RSA moduli are divisible by 8,
+        // since other implementations don't bother.
+        let cert_spki = &cert.tbs_cert.spki;
+        if cert_spki.algorithm.is_rsa() {
+            let rsa_key: Pksc1RsaPublicKey<'_> =
+                asn1::parse_single(cert_spki.subject_public_key.as_bytes())?;
+
+            if rsa_key.n.as_bytes().len() * 8 < self.minimum_rsa_modulus {
+                return Err(ValidationError::Other("RSA key is too weak".into()));
+            }
+        }
```

Comment:
NB: This made me realize that the other key/algorithm checks also probably belong in `permits_*`, not in `valid_issuer` (since CABF doesn't declare keys/algos valid on an issuance basis, but in absolute terms).

I didn't move the other checks in this same PR to keep the diff small, but I can do so (or open a follow-up for them.)